test/functional/ui/output_spec.lua

```diff
@@ -55,7 +55,7 @@ describe("shell command :!", function()
     if 'openbsd' == helpers.uname() then
       pending('FIXME #10804')
     end
-    child_session.feed_data((":!%s REP 30001 foo\n"):format(testprg('shell-test')))
+    child_session.feed_data((":!%s REP 30001 foo\r"):format(testprg('shell-test')))
```

Comment:
I ended up changing the test itself. I mean, `<CR>` means Carriage Return. So `\r` seems more appropriate, right? I donno more in-depth technicalities of these end-of-line stuff, tho.

Reply:
yes that's fine.